properties.schema

```diff
@@ -211,7 +421,61 @@
               "inputType": "TextArea",
               "validators": [],
               "help": "Partly correct answer feedback for any attempt apart from the last attempt. If you leave this blank, the 'Partly Correct Final' feedback will be used instead.",
-              "translatable": true
+              "translatable": true,
+              "properties": {
```

Comment:
I think something's gone a bit wrong here, you've got a `notFinal` property as a child of a `notFinal` property...